Dockerfile

```diff
@@ -18,14 +20,18 @@ ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
-ENV JAVA_HOME=/usr/java/default
-ARG JAVA_PKG
-ADD $JAVA_PKG /usr/java/
-
-RUN export JAVA_DIR=$(ls -1 -d /usr/java/*) && \
-    ln -s $JAVA_DIR /usr/java/latest && \
-    ln -s $JAVA_DIR /usr/java/default && \
-    alternatives --install /usr/bin/java java $JAVA_DIR/bin/java 20000 && \
-    alternatives --install /usr/bin/javac javac $JAVA_DIR/bin/javac 20000 && \
-    alternatives --install /usr/bin/jar jar $JAVA_DIR/bin/jar 20000 && \
-    alternatives --install /usr/bin/keytool keytool $JAVA_DIR/bin/keytool 20000
+RUN set -eux; \
```

Comment:
Have we lost the keytool utility by doing this?

Reply:
no, it's part of the jdk rpm